docs/en/rst/installing/mariadb.rst

```diff
@@ -0,0 +1,124 @@
+.. _mariadb:
+
+MariaDB
+#######
+
+It is strongly advised to use MariaDB which is fully compatible with Bugzilla.
+
+The minimum required version is MariaDB 10.0.5.
+
+It's possible to test which version of MariaDB you have installed with:
+
+:command:`mariadb -e 'select version()'`
```

Comment:
The `mariadb` command is only available in MariaDB 10.4.6 and later. Prior to that you had to use `mysql`. Since we support 10.0.5 and up, we'll need to account for that in the docs. https://mariadb.com/kb/en/mariadb-command-line-client/